### PR TITLE
[Mellanox] Fix 'sensors.conf' mapping for MSN4700

### PR DESCRIPTION
**Pull request by shlomibitton**
**- Why I did it**
Fix 'sensors.conf' mapping for SN4700

**- How I did it**
Edit the 'sensors.conf'.

**- How to verify it**
Run 'sensors' command on Mellanox MSN4700 platform.

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by shlomibitton** [comment]
@keboliu @liat-grozovik @nazariig 
please review

**Comment by lguohan** [comment]
retest vs please

**Comment by liat-grozovik** [comment]
Please dont merge yet. Need to merge first hw mgmt V.7.0000.3034. waiting for PR https://github.com/Azure/sonic-linux-kernel/pull/133 to be merged and taken first to sonic-buildimage

**Comment by lguohan** [comment]
#133 is for 201911 branch, please check.

**Comment by liat-grozovik** [comment]
> #133 is for 201911 branch, please check.

correct. once the hw mgmt will be in both master and 201911 (different PRs) i will merge this one and will tag it for 201911 as well.
I will handle it. thanks!

File: device/mellanox/x86_64-mlnx_msn4700-r0/sensors.conf [hunk 1]
```diff
@@ -21,99 +21,135 @@ bus "i2c-15" "i2c-1-mux (chan_id 6)"
...
+        label in3 "PMIC-4 ASIC 0.85V VCORE_T6_7 Rail (out)"
+        label in4 "PMIC-4 ASIC 1.8V T6_7 Rail (out)"
         label temp1 "PMIC-4 Temp 1"
         label temp2 "PMIC-4 Temp 2"
-        label power1 "PMIC-4 ASIC 0.85V VCORE_T2_3 Rail Pwr (out)"
-        label power2 "PMIC-4 ASIC 1.8V T2_3 Rail Pwr (out)"
-        label curr1 "PMIC-4 ASIC 0.85V VCORE_T2_3 Rail Curr (out)"
-        label curr2 "PMIC-4 ASIC 1.8V T2_3 Rail Curr (out)"
-    chip "tps53679-i2c-*-6a"
-        label in1 "PMIC-5 PSU 12V Rail (in)"
```
Comment (nazariig) [inline device/mellanox/x86_64-mlnx_msn4700-r0/sensors.conf:66]: @shlomibitton why PSU 12V rail has gone?

**Comment by abdosi** [comment]
@liat-grozovik is it ok to cherry-pick in 201911 ?

**Comment by liat-grozovik** [comment]
Yes
Thanks

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Pull request merged**